code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -24,7 +24,8 @@
 	if(!istype(item, /obj/item/skeleton_key) || spawned_loot)
 		return FALSE
 	spawned_loot = TRUE
-	qdel(item)
+	if(!GLOB.magical_access) // you should be capable of opening the chest without a skeleton key if we have magical access, but let miners should buy a key at least.
+		qdel(item)
```

Comment:
This will still require a skeleton key to open the chest, it just won't consume it

Reply:
![image](https://user-images.githubusercontent.com/87972842/217709026-53fe8ae9-4694-4032-a6f0-b66d1a716fd4.png)
changed now